docs/development_testing.md

```diff
@@ -18,7 +18,9 @@ It might be required that you install additional packages depending on your dist
 - Fedora: `sudo dnf -y install systemd-devel gcc-c++`
 - Ubuntu: `sudo apt-get install libudev-dev`
 
-**Important:** Run `tox` before you create your pull request to avoid annoying fixes.
+**Important:** Run `tox` before you create your pull request to avoid annoying fixes. 
+
+**Note:** Running the full `tox` test suite will take quite some time, so as the minimal requirement for pull requests, run at least the tests that are related to your code changes (see details below on how to). The full test suite will anyway be run by the CI once you created your pull request and before it can be merged.
```

Comment:
We should maybe not make this a note, but just incorporate this into the guide.

Reply:
Not sure I get what you mean. "This guide" is this page I edited or something else?

If it is this one: I made a visually distinct note by design, to make sure it is not missed. Otherwise people see the upmost `tox` section and then run it and...have to wait half an hour...and then rant in the forum, Discord, ...

Reply:
The point is, not everything has to be a note. It can be just part of the regular text.

Reply:
I do agree in general, but this changes' purpose is to make that info really easy to see to prevent confusion. If it were just some info about the internal working of `tox` (as the paragraphs after it are), then I would never have marked it as a note 😄.

But if I am the only one with this opinion then I will revert back to a regular text block.

Reply:
![image](https://user-images.githubusercontent.com/195327/111356604-d14b9d80-8688-11eb-8fff-f23c1c957614.png)

I think the use of the lighter note, as suggested by Tom, makes it acceptable.